packages/exchange/src/exchange/providers/azure.py

```diff
@@ -8,8 +8,8 @@ class AzureProvider(OpenAiProvider):
     """Provides chat completions for models hosted by the Azure OpenAI Service."""
 
     PROVIDER_NAME = "azure"
+    BASE_URL_ENV_VAR = "AZURE_CHAT_COMPLETIONS_HOST_NAME"
     REQUIRED_ENV_VARS = [
-        "AZURE_CHAT_COMPLETIONS_HOST_NAME",
```

Comment:
I separated out the base URL enforcement from the other ENV vars as there is special handling